.travis.yml

```diff
@@ -38,12 +38,17 @@ install:
   - conda install --quiet --file conda-requirements.txt
   - pip install coveralls
 
+  # Work around conda's Pillow package lacking ImageCms
+  - conda uninstall pillow
```

Comment:
and then here prefix the two commands here with `[[ "$REINSTALL_PILLOW" == "yes" ]] && `